src/sentry/runner/commands/createuser.py

```diff
@@ -68,23 +80,31 @@ def createuser(email, password, superuser, no_password, no_input, force_update):
         if superuser is None:
             superuser = _get_superuser()
 
+        # for self hosted to give superusers admin permissions
+        if superuser and settings.SENTRY_SELF_HOSTED and superadmin is None:
+            superadmin = _get_superadmin()
+
+        if staff is None:
+            staff = _get_staff()
+
     if superuser is None:
         superuser = False
 
+    if staff is None:
+        staff = superuser
+
```

Comment:
I think we also will need a `if superadmin is None:` block here if we end up going with None-able `--superadmin`.